base/bucket_gocb.go

```diff
@@ -147,6 +152,72 @@ func (bucket CouchbaseBucketGoCB) GetBucketCredentials() (username, password str
 	return bucket.spec.BucketName, password
 }
 
+func (bucket CouchbaseBucketGoCB) GetMetadataPurgeInterval() (int, error) {
```

Comment:
Can you put some comments on the top of this method?  In particular to differentiate between `RetrievePurgeInterval()` behavior, that has a similar name